libs/api/windows_platform.cpp

```diff
@@ -13,83 +13,60 @@
 #include "windows_platform.hpp"
 #include "windows_platform_common.hpp"
 
-int
-create_map_internal(
-    uint32_t type,
-    uint32_t key_size,
-    uint32_t value_size,
-    uint32_t max_entries,
-    uint32_t inner_map_fd,
-    size_t section_offset,
-    ebpf_verifier_options_t options);
-
-static int
-create_map_windows(
-    uint32_t map_type,
-    uint32_t key_size,
-    uint32_t value_size,
-    uint32_t max_entries,
-    uint32_t inner_map_fd,
-    size_t section_offset,
-    ebpf_verifier_options_t options)
-{
-    int fd;
-    if (options.mock_map_fds) {
-        EbpfMapType type = get_map_type_windows(map_type);
-        fd = create_map_crab(type, key_size, value_size, max_entries, options);
```

Comment:
When verifier calls the Windows callback function `parse_maps_section_windows()` and passes `mock_map_fds` option in the verifier options, this function used to call `create_map_crab()` which used to then populate some TLS in the verifier code.
Do we not need that flow now?

Reply:
Nope, the flow is simplified so it's done differently.  The old flow was pretty convoluted.